pkg/operator/validations.go

```diff
@@ -34,3 +35,27 @@ func ValidateDurationField(durationField string) error {
 	}
 	return nil
 }
+
+// CompareScrapeTimeoutToScrapeInterval validates value of scrapeTimeout based on scrapeInterval
+func CompareScrapeTimeoutToScrapeInterval(scrapeTimeout, scrapeInterval string) error {
+	var err error
+	var si, st model.Duration
+
+	// since default scrapeInterval set by operator is 30s
+	if scrapeInterval == "" {
+		scrapeInterval = "30s"
+	}
```

Comment:
it's only the default for the Prometheus CRD so I don't think it should be checked here.
```suggestion
```

Reply:
The default value is set here https://github.com/prometheus-operator/prometheus-operator/blob/5f72fbc655b416b9b50ede04ed2c0a64cda22933/pkg/prometheus/promcfg.go#L339 which comes after this validation so we need to check here right?